### PR TITLE
Add external tests.

### PR DESCRIPTION
**Pull request by chriseth**
Run the zeppelin tests on the current solidity version as part of the emscripten build.

### CODE REVIEW

File: test/externalTests.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+DIR=$(mktemp -d)
+(
+    cd "$DIR"
+    echo "Running Zeppelin tests..."
+    git clone https://github.com/OpenZeppelin/zeppelin-solidity.git
+    cd zeppelin-solidity
+    npm install
+    cp "$SOLJSON" ./node_modules/solc/soljson.js
+    ./scripts/test.sh
```
Comment (axic) [inline test/externalTests.sh]: `npm run test`?

File: scripts/test_emscripten.sh [hunk 2]
```diff
@@ -51,3 +51,6 @@ npm version $VERSION
 
 echo "Running solc-js tests..."
 npm run test
+
+echo "Running external tests...."
+"$REPO_DIR"/test/externalTests.sh "$REPO_DIR"/build/solc/soljson.js
```
Comment (axic) [inline scripts/test_emscripten.sh]: `REPO_DIR`? There's only `REPO_ROOT` in this script.

Could also create a variable for this up next to REPO_ROOT and use it in the "replacing soljson.js" part too.

Comment (chriseth) [reply scripts/test_emscripten.sh]: Not really sure what you mean.

**Comment by axic** [comment]
This finally works. Adds 5 minutes to the Emscripten travis time.

**Review by axic — APPROVED** [review]

**Pull request merged**